### PR TITLE
Feature/maintenance update

### PR DESCRIPTION
**Pull request by meganrm**
I updated the setup of this template to reflect some of the changes we've made through development on simularium.

### CODE REVIEW

File: .github/workflows/auto-deploy-staging.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: Auto deploy staging AWS
+env: 
+  NODE_VERSION: '12.x'
```
Comment (toloudis) [inline .github/workflows/auto-deploy-staging.yml]: when can we get rid of this node version requirement? or at least update it

Comment (meganrm) [reply .github/workflows/auto-deploy-staging.yml]: you want it updated to 14? I think we always need to have it

File: README.md [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
+## Installation
 
-Differences in builds by environment:
+Describe how to obtain the software and get it ready to run
 
-| Target | Sources Maps |  NODE_ENV === 'production' |
-| ------ | ------------ |  ------------------------- |
-| dev    | true         |  false                     |
-| staging| true         |  true                      |
-|production| false      |  true                      |
```
Comment (toloudis) [inline README.md:25]: is the above info no longer true for this project?

File: package.json [hunk 3]
```diff
@@ -1,59 +1,95 @@
...
+  "scripts": {
+    "start": "webpack-dev-server --config ./webpack/webpack.config.js --env.env dev",
+    "gh-build": "GH_BUILD=true webpack --config ./webpack/webpack.config.js --env.env production",
+    "pregh-deploy": "npm run gh-build",
+    "gh-deploy": "gh-pages -d dist",
+    "build:staging": "NODE_ENV=staging webpack --config ./webpack/webpack.config.js  --env.env staging",
+    "build": "NODE_ENV=production webpack --config ./webpack/webpack.config.js  --env.env production",
+    "lint": "eslint src/**/*.{ts,tsx}",
+    "typeCheck": "tsc -p tsconfig.json --noEmit",
+    "update-viewer": "npm i @aics/simularium-viewer@latest",
```
Comment (toloudis) [inline package.json]: update-viewer doesn't belong.  also what is gh-build supposed to do?

File: .github/workflows/auto-deploy-staging.yml (hunk 1 above)
Comment (toloudis) [reply .github/workflows/auto-deploy-staging.yml]: I'd rather just let it be whatever github-actions uses by default, unless we have some compatibility reason to hold it back.

Comment (toloudis) [reply .github/workflows/auto-deploy-staging.yml]: it would also be cleaner if we could define it once and include it in all the other scripts, but I'm not sure how to do that with gh-actions/yaml

Comment (meganrm) [reply .github/workflows/auto-deploy-staging.yml]: i'm pretty sure you always have to define what node version you're using. Also our local dev env dont auto update to the next node or anything so I think it's good to know it's the same rather than have the remote sometime using a newer version

File: package.json (hunk 3 above)
Comment (meganrm) [reply package.json]: i removed it and gh-build. gh-deploy will now just do a regular build by default and push it to the gh-pages branch

File: README.md (hunk 2 above)
Comment (meganrm) [reply README.md:25]: I moved it into the CONTRIBUTING doc

**Review by toloudis — APPROVED** [review]

File: src/containers/App/index.tsx [hunk 4]
```diff
@@ -6,7 +6,7 @@ import Sidebar from "../Sidebar";
 
 const styles = require("./style.css");
 
-export default class App extends React.Component<{}, {}> {
+export default class App extends React.Component<unknown, never> {
```
Comment (schoinh) [inline src/containers/App/index.tsx:9]: Just curious, why are these changes necessary?

File: src/styles/ant-vars.less [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+@white-six: #e7e7e7;
+@baby-purple: #b59ff6;
+@dark-purple: #3b3649;
+@allen-purple: #8d87aa;
+@greyish-brown: #4a4a4a;
+@highlight-green: #b2d030;
+@blue: #0094FF;
+
+// -------- Colors -----------
+@primary-color : @baby-purple;
```
Comment (schoinh) [inline src/styles/ant-vars.less:17]: I like that this .less file is included in this template, but I wonder if it would be better start it off from a blank slate? I think a lot of the styling in here is specific to simularium-website. What I'm envisioning is a copy of https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less that is entirely commented out

**Review by schoinh — COMMENTED** [review]
Wow, thanks for doing this update!

File: src/containers/App/index.tsx (hunk 4 above)
Comment (meganrm) [reply src/containers/App/index.tsx:9]: I got a new linting error that says `{}` shouldnt be used as a type and then gave different suggestions for if you just don't know the type verses there will never be a value.

File: src/styles/ant-vars.less (hunk 5 above)
Comment (meganrm) [reply src/styles/ant-vars.less:17]: that's a good thought. The reason I put this here is because it seems like the simularium branding might become the Animated Cell branding, so we'd end up copying over a lot of color settings anyway? I dunno which is more likely

Comment (toloudis) [reply src/styles/ant-vars.less:17]: I'm not sure how much we will even be using this project repo, but I imagined this file as sort of the AICS web front end style guide, containing our stock color palette etc.  It would be a starting point for further changes in specific projects

Comment (schoinh) [reply src/styles/ant-vars.less:17]: Ok, I'm ok with this too :)

**Review by schoinh — APPROVED** [review]
